aiosmtpd/controller.py

```diff
@@ -425,7 +425,7 @@ def _trigger_server(self):
         with ExitStack() as stk:
             s = stk.enter_context(create_connection((hostname, self.port), 1.0))
             if self.ssl_context:
-                s = stk.enter_context(self.ssl_context.wrap_socket(s))
+                s = stk.enter_context(self.ssl_context.wrap_socket(s, server_side=True))
```

Comment:
I'm staring at this for 15 minutes now and I can't make the sense out of it. The code is opening a *client connection* here, so why would it use server-side context?

Reply:
Yeah, both of these are correct interpretations. Also reading the surrounding comments, the goal here is to connect to the server so that whatever server init stuff needs to happen, happens. I'm pretty sure that the author is seeing some behavior probably due to misconfigured SSL. I need to go double check the original issue.